proposals/NNNN-POI-Service-Supported.md

```diff
@@ -3,62 +3,61 @@
 * Proposal: [SDL-NNNN](NNNN-POI-Service-Supported.md)
 * Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
 * Status: Awaiting review
-* Impacted Platforms: [ Core / iOS / Java Suite / RPC ]
+* Impacted Platforms: [ Core / RPC ]
 
 ## Introduction
 
-This porposal provide the function of sharing POI data between SDL App by `AppService`.  
+This proposal provides the function of sharing POI data between SDL Apps by `AppService`.  
 
 
 ## Motivation
 
 Recently, IVI has moved from AVN (Audio & Visual Navigation) to DA (Display Audio).  
 Therefore, when there is no Native Navigation, it is necessary to share POIs between SDL apps.  
-The proposal of [SDL-0241 Commerce App Service](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0241-Commerce-App-Service.md) included a function to provide search data to other apps,   
-but there was no specific use case and the function was excessive. In this proposal,   
-only the minimum necessary POI exchange is provided for the App service.  
-In the future, if someone want to increase the data exchanged between apps,   
+The proposal of [SDL-0241 Commerce App Service](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0241-Commerce-App-Service.md) included a function to provide search data to other apps, but there was no specific use case and the function was excessive.  
+In this proposal, only the minimum necessary POI exchange is provided for the App service.  
```

Comment:
Please change this line to the following:
>In this proposal, only the minimum necessary POI exchange is provided for the App service. In the future, if someone wants to increase the data exchanged between apps, it is assumed that proposals and extensions will be made each time.  

You can then remove lines 19 and 20.  This will help the section to be formatted in a more readable way.

Reply:
modified!

commit 992b7e3435acf4ed2391f6c059b8cc8d5730b62a (HEAD -> master, origin/master, origin/HEAD)
Author: 河野 祥平 <103082@ad.nexty-ele.com>
Date:   Wed Dec 11 13:34:40 2019 +0900

    Remove unnecessary break